PyInstaller/utils/hooks/__init__.py

```diff
@@ -786,7 +786,6 @@ def collect_system_data_files(path, destdir=None, include_py_files=False):
     # which may not be true on Windows; Windows allows Linux path separators in
     # filenames. Fix this by normalizing the path.
     path = os.path.normpath(path)
-    path = os.path.dirname(path)
```

Comment:
This probably slipped in from some other changeset. But it would be caught anyway due to merge conflict.

Reply:
Hmm yeah this slipped in when I updated the branch. Weird. I'll drop it.